Kitodo/src/test/java/org/kitodo/production/helper/tasks/NewspaperMigrationTaskIT.java

```diff
@@ -0,0 +1,297 @@
+                    + "<mets:file ID=\"FILE_0005\" MIMETYPE=\"\"><mets:FLocat LOCTYPE=\"URL\" xlink:href=\"" + media
+                    + "00000005.tif\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"/></mets:file>"
+                    + "</mets:fileGrp></mets:fileSec>"
+                    + "<mets:structMap TYPE=\"LOGICAL\"><mets:div ID=\"LOG_0007\" TYPE=\"Newspaper\">"
+                    + METS_MPTR_LOCTYPE_URL_XLINK_HREF_XMLNS_XLINK_HTTP_WWW_W3_ORG_1999_XLINK
+                    + "<mets:div ID=\"LOG_0008\" TYPE=\"NewspaperYear\">"
+                    + METS_MPTR_LOCTYPE_URL_XLINK_HREF_XMLNS_XLINK_HTTP_WWW_W3_ORG_1999_XLINK
+                    + "<mets:div ID=\"LOG_0009\" TYPE=\"NewspaperMonth\"><mets:div ID=\"LOG_0010\" TYPE=\"NewspaperDay\">"
+                    + "<mets:div DMDID=\"DMDLOG_0006\" ID=\"LOG_0011\" TYPE=\"PeriodicalIssue\"/>"
+                    + "</mets:div></mets:div></mets:div></mets:div></mets:structMap>"
+                    + "<mets:structMap TYPE=\"PHYSICAL\"><mets:div DMDID=\"DMDPHYS_0000\" ID=\"PHYS_0000\" TYPE=\"BoundBook\">"
+                    + "<mets:div ID=\"PHYS_0001\" ORDER=\"1\" ORDERLABEL=\"uncounted\" TYPE=\"page\">"
+                    + "<mets:fptr FILEID=\"FILE_0001\"/></mets:div>"
+                    + "<mets:div ID=\"PHYS_0002\" ORDER=\"2\" ORDERLABEL=\"uncounted\" TYPE=\"page\">"
+                    + "<mets:fptr FILEID=\"FILE_0002\"/></mets:div>"
+                    + "<mets:div ID=\"PHYS_0003\" ORDER=\"3\" ORDERLABEL=\"uncounted\" TYPE=\"page\">"
+                    + "<mets:fptr FILEID=\"FILE_0003\"/></mets:div>"
+                    + "<mets:div ID=\"PHYS_0004\" ORDER=\"4\" ORDERLABEL=\"uncounted\" TYPE=\"page\">"
+                    + "<mets:fptr FILEID=\"FILE_0004\"/></mets:div>"
+                    + "<mets:div ID=\"PHYS_0005\" ORDER=\"5\" ORDERLABEL=\"uncounted\" TYPE=\"page\">"
+                    + "<mets:fptr FILEID=\"FILE_0005\"/></mets:div></mets:div></mets:structMap><mets:structLink>"
+                    + "<mets:smLink xlink:to=\"PHYS_0001\" xlink:from=\"LOG_0011\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"/>"
+                    + "<mets:smLink xlink:to=\"PHYS_0002\" xlink:from=\"LOG_0011\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"/>"
+                    + "<mets:smLink xlink:to=\"PHYS_0003\" xlink:from=\"LOG_0011\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"/>"
+                    + "<mets:smLink xlink:to=\"PHYS_0004\" xlink:from=\"LOG_0011\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"/>"
+                    + "<mets:smLink xlink:to=\"PHYS_0005\" xlink:from=\"LOG_0011\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"/>"
+                    + "</mets:structLink></mets:mets>");
+        FileUtils.writeLines(new File(processHome, "meta.xml"), "UTF-8", lines);
+    }
+
+    @Test
+    public void testNewspaperMigrationTask() throws Exception {
+        NewspaperMigrationTask underTest = new NewspaperMigrationTask(batchService.findById(5));
```

Comment:
please add assertions before you start the task.
usually they are the same assertions like afterwards but with different "expected" values